code/WorkInProgress/pomf/spacepods/spacepods.dm

```diff
@@ -216,15 +216,15 @@
 		if("pod_civ")
 			desc = "A sleek civilian space pod."
 		if("pod_black")
-			desc = "An all black space pod with no insignias."
+			desc = "A space pod painted black."
```

Comment:
this is a case of a description that might as well not be there

why are you replacing descriptions with something that might as well not be there